### PR TITLE
Ocrvs 2977 Languages Fix

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

File: src/features/languages/generated/client/client.json [hunk 1]
```diff
@@ -861,6 +868,8 @@
         "form.field.label.childDateOfBirth": "Date of birth",
         "form.field.label.childFamilyName": "Last name",
         "form.field.label.childFirstNames": "First name(s)",
+        "form.field.label.childFamilyNameEng": "Last Name(s) in English",
+        "form.field.label.childFirstNamesEng": "First Name(s) in English",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json]: This is not right.  Why do we need this @sadmananik ?

File: src/features/languages/generated/client/client.json [hunk 2]
```diff
@@ -1574,7 +1583,7 @@
         "review.form.section.review.title": "Review",
         "review.formData.header": "{isDraft, select, true {Check responses with the informant before sending for review} other {Review the answers with the supporting documents}}",
         "review.header.subject.subjectWithoutName": "{eventType, select, birth {Birth} other {Death}} declaration",
-        "review.header.subject.subjectWitName": "{eventType, select, birth {Birth} other {Death}} declaration for {name}",
+        "review.header.subject.subjectWitName": "{eventType, select, birth {Birth} other {Death}} Declaration for {name}",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json]: Why do we need to capitalise this @sadmananik

File: src/features/languages/generated/client/client.json [hunk 3]
```diff
@@ -1431,7 +1440,7 @@
         "recordAudit.noInformant": "No informant",
         "recordAudit.noName": "No name provided",
         "recordAudit.history.started":"Started",
-        "recordAudit.history.reinstated.prefix":"Reinstated to ",
+        "recordAudit.history.reinstated":"Reinstated to {status}",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json:1443]: Whats wrong with the key?

File: src/features/languages/generated/client/contentful-keys-to-migrate.json [hunk 4]
```diff
@@ -20,6 +20,8 @@
   "constants.archived_declaration",
   "duplicates.warning",
   "error.passwordSubmissionError",
+  "location.jurisdictionType.UNION",
+  "location.jurisdictionType.DIVISION",
```
Comment (euanmillar) [inline src/features/languages/generated/client/contentful-keys-to-migrate.json]: Why do we need to add these Bangladesh requirements back in?  It is technical debt @sadmananik

File: src/features/languages/generated/client/descriptions.json [hunk 5]
```diff
@@ -231,6 +237,8 @@
     "search.waitingForConnection": "The text when there is no connectivity",
     "error.page.load.failed": "Error message when page failed to load",
     "error.passwordSubmissionError": "Error message when password verification fails",
+    "location.jurisdictionType.UNION": "Label for Union",
```
Comment (euanmillar) [inline src/features/languages/generated/client/descriptions.json]: As above ... technical debt

File: src/features/languages/generated/client/client.json (hunk 3 above)
Comment (Zangetsu101) [reply src/features/languages/generated/client/client.json:1443]: The key's been changed in the core PR as well as we don't really need the _prefix_ part here

File: src/features/languages/generated/client/contentful-keys-to-migrate.json (hunk 4 above)
Comment (Zangetsu101) [reply src/features/languages/generated/client/contentful-keys-to-migrate.json]: Yeah we don't need them in farajaland and can be removed from core as well after refactoring the unit tests

Comment (sadmananik) [reply src/features/languages/generated/client/contentful-keys-to-migrate.json]: These were generated auto while commiting.

File: src/features/languages/generated/client/descriptions.json (hunk 5 above)
Comment (sadmananik) [reply src/features/languages/generated/client/descriptions.json]: Same. Auto generated

File: src/features/languages/generated/client/client.json (hunk 2 above)
Comment (sadmananik) [reply src/features/languages/generated/client/client.json]: Fixed it.

File: src/features/languages/generated/client/client.json (hunk 1 above)
Comment (sadmananik) [reply src/features/languages/generated/client/client.json]: Removed it.

File: src/features/languages/generated/client/client.json [hunk 6]
```diff
@@ -397,6 +399,10 @@
         "constants.reason": "Reason",
         "constants.registerConfirmModalDesc": "",
         "constants.registered": "Registered",
+        "constants.inReview.status": "In Review",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json]: In review

File: src/features/languages/generated/client/client.json [hunk 7]
```diff
@@ -1604,6 +1611,9 @@
         "search.searchResultFor": "{total, plural, =0 {No results for “{param}”} other {Search results for “{param}”}}",
         "search.totalResultText": "{total, plural, =0 {} one {# record found} other {# records found}} ",
         "search.waitingForConnection": "Reconnect to load declarations",
+        "secureAccount.page.title": "Secure your Account",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json]: Secure your account

File: src/features/languages/generated/client/client.json [hunk 8]
```diff
@@ -1665,6 +1675,11 @@
         "system.user.settings.phonedNumberUpdated": "Phone number updated",
         "system.user.settings.profileImage": "Profile Image",
         "system.user.duplicateMobileError": "{number} is already used by another user. Please use a different phone number",
+        "system.user.newUser": "New User",
```
Comment (euanmillar) [inline src/features/languages/generated/client/client.json]: New user

File: src/features/languages/generated/client/client.json (hunk 7 above)
Comment (sadmananik) [reply src/features/languages/generated/client/client.json]: Fixed

File: src/features/languages/generated/client/client.json (hunk 6 above)
Comment (sadmananik) [reply src/features/languages/generated/client/client.json]: Fixed

File: src/features/languages/generated/client/client.json (hunk 8 above)
Comment (sadmananik) [reply src/features/languages/generated/client/client.json]: Fixed

**Pull request merged**